README.md

```diff
@@ -251,8 +253,7 @@ pact.verifyPacts({
 | Parameter                   | Required? | Type    | Description                                                                                                |
 | --------------------------- | --------- | ------- | ---------------------------------------------------------------------------------------------------------- |
 | `providerBaseUrl`           | true      | string  | Running API provider host endpoint.                                                                        |
-| `pactBrokerBaseUrl`         | false     | string  | Base URL of the Pact Broker from which to retrieve the pacts.                                                                        |
-| `pactBrokerUrl`             | false     | string  | URL of your Pact Broker to dynamically discover relevent pacts to verify. Required if `pactUrls` not given |
```

Comment:
I've deprecated this field. So whilst it is still supported, I've removed it from the docs here and have provided a warning if it's used.

Reply:
I don't see why it should be deprecated.  They do exactly the same thing no?  Just the name that's different and I did that on purpose since `base` doesn't really add any value.